index.html

```diff
@@ -1736,24 +1718,20 @@
       </section>
       <section>
         <h3>
-          Interface <a><code>PresentationReceiver</code></a>
+          Interface <a><code>PresentationConnectionList</code></a>
```

Comment:
Are we using this for both the controlling and receiving browsing contexts?

We could keep this as a generic interface that could be implemented by either or both, but if we need to add additional features that are specific to the receiving context I feel we would end up defining a PresentationReceiver interface again anyway.